_posts/en/2021-08-20-micro-acquisition-pilot-update2.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "Micro-Acquisition Pilot: Status Update 2"
+ref: micro-acquisition-update2
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team and Micro-Acquisition Pilot team members"
+date: "2021-08-20"
+excerpt: "The Micro-Acquisition pilot is live! In this blog post we share what we have learned so far and how we are adjusting and improving."
+---
+<!--markdownlint-disable MD033-->
+Since our last status update, the [Micro-Acquisition website](https://micro-acquisition.alpha.canada.ca/en/index.html) went live and the [first opportunity](https://micro-acquisition.alpha.canada.ca/2021/06/23/building-custom-Azure-marketplace-policies.html) was posted.
+
+As part of the Micro-Acquisition pilot we are trying things, learning and adjusting as we go.
+In preparation for the next opportunity we have made a few changes based on what we learned from the first opportunity.
+
+## Added a mailing list
+
+When we didn't get a lot of followers on our [Twitter account](https://twitter.com/microbuysgc), we did some testing and realized 2 things:
+
+1. It wasn't clear what information would be found by following the Twitter account and
+2. A lot of our target audience is not on Twitter
+
+We now have a mailing list that anyone can subscribe to.
+Through the [mailing list](https://forms-formulaires.alpha.canada.ca/id/36) (and Twitter) we will be sharing when new opportunities are posted and/or updated. We also updated the text on the opportunities page to be more clear what information is available via the Twitter account and mailing list.
+
+## Updated evaluation question
+
+Our first opportunity was for a less well known language (Powershell).
+We realized from the applications we received that there were people who likely could do the work but the phrasing of our evaluation criteria prevented us from giving them a pass because they didn't have experience specifically in Powershell.
```

Comment:
If the previous note about Powershell should be in a different section, then the focus here is less about Powershell and the way applicants framed their answers, as per the rest of this section.